manticore/manticore.py

```diff
@@ -315,6 +337,18 @@ def verbosity(self, setting):
     def maxstorage(self, max_storage):
         self._maxstorage = max_storage
 
+    @property
+    def workspace(self):
+        return self._workspace
+
+    @workspace.setter
+    def workspace(self, ws):
```

Comment:
Do we want to change the workspace after creation? Is there a case we do not know the workspace at constructor time ?

Reply:
I think that changing the workspace after analysis has started could potentially do some pretty weird things (what if states are already written to that workspace?) and doesn't seem like something a user would need to do.